lib/router.js

```diff
@@ -5,78 +5,167 @@ const express = require('express');
+// enables us to dynamically load multiple models with similiar routes
 router.param('model', modelFinder.load);
 
-// TODO: Swagger Comment
+/**
+ * Homepage route for the site
+ * @route GET /
+ * @returns {string} 200 - A welcome message to show you're on the Homepage
+ */
 router.get('/', (req, res, next) => {
   res.send('Homepage');
 });
 
-// TODO: Swagger Comment
+/**
+ * Route for listing out the available models
+ * @route GET /models
+ * @group Models - Operations performed on the database
+ * @returns {object} 200 - A JSON object of the models in our database
+ */
 router.get('/models', (req, res, next) => {
   modelFinder.list().then(models => res.status(200).json(models));
 });
 
-// TODO: Swagger Comment
+/**
+ * Dynamic route for getting the schema of all models in our database
+ * @route GET /:model/schema
+ * @group Models - Operations performed on the database
+ * @param {string} - :model - the specific model you would like to access
+ * @returns {object} 200 - JSON object containing specific information about a models schema
+ * @returns {Error} 500 - Generic error for incorrect param
+ */
 router.get('/:model/schema', (req, res, next) => {
   res.status(200).json(req.model.jsonSchema());
 });
 
-// TODO: Swagger Comment
+/**
+ * Dynamic route for getting the records of a collection/model in our database
+ * @route GET /:model
+ * @group Models - Operations performed on the database
+ * @param {string} - :model - the specific model you would like to access
+ * @returns {object} 200 - JSON object containing all of the records for a given collection
+ * @returns {Error} 500 - Generic error for incorrect param
+ */
 router.get('/:model', handleGetAll);
 
-// TODO: Swagger Comment
+/**
+ * Dynamic route for creating a record in the database
+ * @route POST /:model
+ * @group Models - Operations performed on the database
+ * @param {string} - :model - the specific model you would like to access
+ * @returns {object} 200 - JSON object containing the created item as it appears in the database
+ * @returns {Error} 500 - Generic error for incorrect param
+ */
 router.post('/:model', handlePost);
 
-// TODO: Swagger Comment
+/**
+ * Dynamic route for getting one record by it's ID
+ * @route GET /:model/:id
+ * @group Models - Operations performed on the database
+ * @param {string} - :model - the specific model you would like to access
+ * @param {string} - :id - the id of the record you are trying to find
+ * @returns {object} 200 - JSON object containing the found record
+ * @returns {Error} 500 - Generic error for incorrect param
+ */
 router.get('/:model/:id', handleGetOne);
 
-// TODO: Swagger Comment
+/**
+ * Dynamic route for finding and updating a record
+ * @route PUT /:model/:id
+ * @group Models - Operations performed on the database
+ * @param {string} - :model - the specific model you would like to access
+ * @param {string} - :id - the id of the record you are trying to update
```

Comment:
Again, having a `@param` to represent `req.body` would be useful in allowing people to understand how to use this route